src/main/java/core/basesyntax/ReturnOperation.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Map;
+
+public class ReturnOperation implements OperationHandler {
```

Comment:
The class name `ReturnOperation` is not quite descriptive. Consider renaming it to more closely reflect its responsibility, such as `ReturnOperationHandler`, to adhere to the naming conventions and clarify its role in the Strategy pattern.